front/components/home/content/Industry/configs/utils.tsx

```diff
@@ -0,0 +1,178 @@
+import type { RocketIcon } from "@dust-tt/sparkle";
+import type { ReactNode } from "react";
+
+// Basic config types
+export interface ChipConfig {
+  label: string;
+  color:
+    | "primary"
+    | "success"
+    | "warning"
+    | "info"
+    | "highlight"
+    | "green"
+    | "blue"
+    | "rose"
+    | "golden";
+  icon?: typeof RocketIcon;
```

Comment:
Let's be a bit more flexible and use the underlying type. IIRC its an `Element`?